packages/mendel-browserify/index.js

```diff
@@ -203,6 +210,35 @@ MendelBrowserify.prototype.doneManifest = function() {
     );
 }
 
+MendelBrowserify.prototype.sortedManifest = function() {
+    var sortedManifest = {
+        indexes: {},
+        bundles: []
+    };
+
+    var self = this;
+    Object.keys(this._manifestIndexes).sort().forEach(function(file) {
+        var bundle = self._manifestBundles[self._manifestIndexes[file]];
+        sortedManifest.bundles.push(bundle);
+
+        var index = sortedManifest.bundles.indexOf(bundle);
```

Comment:
I didn't mean it was wrong :) shoul have been more explicit, i meant that since you just push to the array, then the index is `sortedManifest.bundles.length - 1` right? And you wouldn't need to search the array with indexOf and could arguably be faster...